finite-supply.adoc

```diff
@@ -43,84 +43,84 @@ https://bitcointalk.org/index.php?topic=7253.0[here].
 This leaves us with: *???* BTC.
 ____
 
-We can thus be sure that the money supply is at most
-20999817.31308491 BTC. Any lost or unverifiably burnt coins will make
-this number lower, but we don't know how much lower. The interesting
-thing is that it doesn't really matter, or it does matter in a
+We can thus be sure that the bitcoin supply will be
+20999817.31308491 BTC at most. Any lost or unverifiably burnt coins will make
+this number lower, but we don't know by how much. The interesting
+thing is that it doesn't really matter, or better yet it does matter in a
 positive way for bitcoin holders,
 https://bitcointalk.org/index.php?topic=198.msg1647#msg1647[as
 explained] by Satoshi Nakamoto:
 
-[quote,Satoshi Nakamoto on lost bitcoins, Bitcointalk forum]
+[quote,Satoshi Nakamoto on lost bitcoins, Bitcointalk forum (2010)]
 ____
 Lost coins only make everyone else's coins worth slightly more.  Think
 of it as a donation to everyone.
 ____
 
-The finite supply will shrink which, at least in theory, which should
+The finite supply will shrink and this should, at least in theory,
 cause price deflation.
 
-What's more important than the exact number of coins in circulation is
-how the consensus rule of limited supply is enforced without a central
+More important than the exact number of coins in circulation is
+the way the supply limit is enforced without any central
 authority. Alias chytrik puts it well on
 https://bitcoin.stackexchange.com/a/106830/69518[Stack Exchange].
 
-[quote, chytrik, Stack Exchange]
+[quote, chytrik, Stack Exchange (2021)]
 ____
 So the answer is that you don't have to trust someone to not increase
 the supply. You just have to run some code that will verify that they
 haven't.
 ____
 
-Even if some full nodes give themselves to the dark side and decide to
-accept blocks with higher-value coinbase transactions, all other full
-nodes will simply neglect them and continue as usual. Some full nodes
-may intentionally or unintentionally (see the
-<<combined-output-overflow,When shit hits the fan>> chapter) run bad
-software, yet the collective robustly secures the blockchain
-together. Thus you can choose to trust the system without having to
+Even if some full nodes turn to the dark side and decide to
+accept blocks with higher-value coinbase transactions, all the remaining full
+nodes will simply neglect them and continue doing business as usual. Some full nodes
+may, intentionally or unintentionally (see
+<<combined-output-overflow,When shit hits the fan>>), run evil
```

Comment:
If technical xrefs, use `(see <<combined-output-overflow>>)`

Reply:
I'm not sure what you mean. Should I delete ",When shit hits the fan"?

Reply:
Yes, if we decide to go for technical xrefs instead of descriptive xrefs. See my comment on privacy: https://github.com/bitcoin-dev-philosophy/btcphilosophy/pull/70#discussion_r1094980249